include/zephyr/sys/atomic.h

```diff
@@ -132,7 +132,7 @@ static inline bool atomic_test_bit(const atomic_t *target, int bit)
 }
 
 /**
- * @brief Atomically test and clear a bit.
+ * @brief Atomically clear and test a bit.
```

Comment:
The point I was trying to make here (although a bit implicit), is that the `clear` part is atomic, but the `test` part is not. In other words, I wanted to avoid the phrase "Atomically test"

It is somewhat inspired by the phrasing from https://en.cppreference.com/w/c/atomic/atomic_flag_test_and_set: 

> Atomically changes the state of a atomic_flag pointed to by obj to set (true) and returns the previous value.

Reply:
But anyway no matter how atomic this function is, as was discussed in Discord anything may happen (threads may be preempted) before any more of the code calling this function is run. But I think this is clear enough, and we don't need to discuss this more than that. If you think that changing the phrasing improves its understandability, then it's an improvement.

Though maybe the following phrasing would make the point you are trying to make more clear:
```suggestion
 * @brief Atomically clear a bit and test it.
```

Reply:
> But anyway no matter how atomic this function is, as was discussed in Discord anything may happen (threads may be preempted) before any more of the code calling this function is run. 

I agree. It is purely for semantics for me :) 

> Though maybe the following phrasing would make the point you are trying to make more clear:

Sure I can update that for the set and clear functions. Decided for the current one to match the wording in `atomic_test_bit`